cloudfront/wellcomelibrary.org/edge-lambda/testRedirects.ts

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import fs from 'fs';
+import axios from 'axios';
+import chalk from 'chalk';
+
+type RedirectTest = {
+  host: string;
+  pathTests: Record<string, string>;
+};
+
+type HostEnvs = {
+  prod: string;
+  stage: string;
+};
+
+type EnvId = 'stage' | 'prod';
+
+async function testRedirects(tests: RedirectTest) {
+  Object.entries(tests.pathTests).map(async ([fromPath, to]) => {
+    const from = `${tests.host}${fromPath}`;
+
+    try {
+      const axiosResponse = await axios.get(from);
+      const responseUrl = axiosResponse.request.res.responseUrl;
+      if (responseUrl === to) {
+        console.info(chalk.green(`✓ ${from} === ${responseUrl}`));
```

Comment:
How come these tests aren't using jest? Feels a bit like reinventing the wheel to print these assertion outcomes...

Reply:
It did cross my mind. I'm not sure it makes sense to use jest here as these aren't really testing the lambda, (but they really are?). I'll have a think.

Reply:
It doesn't matter all that much, but I would ask why whether they're testing the lambda should change whether you use a test runner (I'd say it shouldn't).

Reply:
I really don't have a strong feeling about this - other than maybe next time i will _try_ using a test runner for slightly non-conventional tests.